.travis.yml

```diff
@@ -12,7 +12,6 @@ addons:
     - gcc-4.9   # Needed for C++11
     - g++-4.9   # Needed for C++11
     - gdb
-    - gcov
```

Comment:
I removed gcov because travis cannot find the package. I checked on packages.ubuntu.com and a search shows that gcov is part of the gcc package.

I believe this is an already existing issue with travis, which was introduced when dist was changed from trusty to precise in commit 5d129991 (the Arrow page on github shows error as Travis badge)